test.js

```diff
@@ -1,13 +1,14 @@
 import fs from 'fs';
 import path from 'path';
+import globby from 'globby';
 import test from 'ava';
 import execa from 'execa';
 import tempy from 'tempy';
 
 const bin = path.join(__dirname, 'cli.js');
 const fixture = fs.readFileSync(path.join(__dirname, 'fixtures/code-of-conduct.md'), 'utf8');
 
-test('generate', async t => {
+test.serial('generate', async t => {
```

Comment:
Because the tests were interfering with each other due to both being run at the same time and accessing system vars

Reply:
What system variables are they accessing?

Reply:
Basically what is happening is the `generate` and `update` tests are interfering with one another.

While the `generate` test declares the email explicitly with the flag, the `update` test doesn't. The side effect of this is that the email in the `update` test is either the one specified in `generate` or the travis email address (as its set in the git config), though I can't exactly remember which.

An example of this can be seen with previous test runs for this PR.

Reply:
Ok, makes sense.